src/ytdl_sub/plugins/static_nfo_tags.py

```diff
@@ -7,10 +7,11 @@
 
 class StaticNfoTagsOptions(SharedNfoTagsOptions):
     """
-    Adds an NFO file for every entry, but does not link it to an entry in the download archive.
```

Comment:
What is the change here?

Reply:
[To satisfy the test](https://github.com/jmbannon/ytdl-sub/pull/1282#issuecomment-3193006117).